lms/static/sass/views/_homepage.scss

```diff
@@ -0,0 +1,168 @@
+// lms - views - homepage view
+// ====================
+
+$course-card-height: ($baseline*18);
+$course-image-height: ($baseline*8);
+$course-info-height: ($baseline*10);
+$course-title-height: ($baseline*3.6);
+$learn-more-horizontal-position: calc(50% - 100px); // calculate the left position for "LEARN MORE" content
+
+.courses-container {
+  @include outer-container;
+
+  .courses {
+    @include clearfix();
```

Comment:
Please remove the ``clearfix()`` mixin reference here and replace it with the Neat-based ``row()`` mixin (http://thoughtbot.github.io/neat-docs/latest/#row).